From 7579b0d65d3c68839c06e70dec5330c7d9b62e2f Mon Sep 17 00:00:00 2001
From: Shuah Khan <shuahkh@osg.samsung.com>
Date: Fri, 9 Jun 2017 11:28:18 -0600
Subject: [PATCH 19/50] kselftest: MAINTAINERS git tree entry update files and
 dirs

Add missing trailing slash to tools/testing/selftests to cover all files
and directories below. Add kselftest documentation files.

Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>
---
 MAINTAINERS | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index f3bb488..c0abc2f 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7246,7 +7246,8 @@ M:	Shuah Khan <shuah@kernel.org>
 L:	linux-kselftest@vger.kernel.org
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/shuah/linux-kselftest.git
 S:	Maintained
-F:	tools/testing/selftests
+F:	tools/testing/selftests/
+F:	Documentation/dev-tools/kselftest*
 
 KERNEL VIRTUAL MACHINE (KVM)
 M:	Paolo Bonzini <pbonzini@redhat.com>
-- 
2.7.4

